src/Avalonia.Base/Controls/IPseudoClasses.cs

```diff
@@ -19,5 +19,12 @@ public interface IPseudoClasses
         /// </summary>
         /// <param name="name">The pseudoclass name.</param>
         bool Remove(string name);
+
+        /// <summary>
+        /// Returns whether a pseudoclass is present in the collection.
+        /// </summary>
+        /// <param name="name">The pseudoclass name.</param>
+        /// <returns>Whether the pseudoclass is present.</returns>
+        bool Has(string name);
```

Comment:
This is called `Contains` on `Classes` (and all other collections) - any reason for the different naming here?